### ISSUE TITLE
Invalid value for word-wrap

### ISSUE THREAD

**Issue by variadico** [johnotander/furtive#42]
Hi, there.

So, I noticed this rule at the end of `furtive.css`.

````
table th, table td {
  padding: 0.25rem;
  word-wrap: none;
  line-height: 1; }
````

What is `word-wrap: none;` supposed to do? According to MDN and W3C, `none` isn't a valid value for `word-wrap`.

https://developer.mozilla.org/en-US/docs/Web/CSS/word-wrap
http://dev.w3.org/csswg/css-text-3/#propdef-word-wrap

Thanks! :smiley:

**Comment by johnotander** [comment]
To be honest, I can't remember when writing that. I imagine I was aiming for `word-wrap: normal;`.

**Comment by pi241a** [comment]
Ha, okay. I'll assume you meant `normal`.

**Comment by johnotander** [comment]
Resolved with #46.
